stages/org.osbuild.sysconfig

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python3
+"""
+
+Configure sysconfig files
+
+The sysconfig directory contains a variety of system configuration files. The
+values found in each of the configuration files may need to be specified
+before first boot.
+
+Currently, only the kernel and network configs can be modified.
```

Comment:
Could elaborate what options are supported and what type have. I wonder if there is a good manual to link to.

Reply:
I would love a good manual haha. The best I came across was [this one for](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/ch-the_sysconfig_directory) rhel 6 but it doesn't really have that much info and doesn't cover our NOZEROCONF setting.